lib/shared/setup-github-actions.ts

```diff
@@ -32,7 +32,7 @@ jobs:
       - uses: actions/checkout@v4
       - uses: oven-sh/setup-bun@v2
       - run: bun install
-      - run: bunx tsci build
+      - run: bunx tsci build --ignore-errors
```

Comment:
idk i think it's a good idea to generally force circuits to always build without errors

```suggestion
      - run: bunx tsci build
```